internal/c/libqb/gui.cpp

```diff
@@ -525,7 +525,18 @@ void sub__glrender(int32 method){
             
             
             if (render_state.dest_handle==0){
-                static int32 dst_w,dst_h;
+                static int32 dst_w,dst_h;
+                static int32 scale_factor=0;
+
+                #ifdef QB64_MACOSX
+                    if (scale_factor==0) {
+                        scale_factor=1;
+                        if (system("system_profiler SPDisplaysDataType | grep Retina")==0) scale_factor=2;
```

Comment:
I doubt this is the correct way of detecting the correct scaling factor. 

It likely works when all screens (one or more) are "Retina", but it will very likely fail when one has say, Retina screen on their notebook and a non-retina screen attache to that.

Reply:
We're together in doubting it. It'll fail with more than one monitor, for sure. So far what we got is complaints from users with single Retina monitors and till a better approach comes up, that shall do the trick.